frontend/src/components/common/Resource/ResourceTable.tsx

```diff
@@ -458,6 +462,23 @@ function ResourceTableContent<RowItem extends KubeObject>(props: ResourceTablePr
     };
   }, [actionsProcessed]);
 
+  const wrappedEnableRowSelection = useMemo(() => {
+    console.log(import.meta.env);
+    if (import.meta.env.REACT_APP_HEADLAMP_ROW_SELECTION_DISABLED === 'true') {
```

Comment:
I'd prefer `...ENABLE_ROW_SELECTION`. If present and `'false'` -> disable. Otherwise, enabled.